app-dicts/dictd-devils/dictd-devils-20150822.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The Devil's Dictionary for dict"
+HOMEPAGE="http://www.dict.org/"
+SRC_URI="http://www.gutenberg.org/files/972/972.zip -> $P.zip"
+
+LICENSE="public-domain"
+SLOT="0"
+IUSE=""
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86"
+
+BDEPEND=">=app-text/dictd-1.5.5 \
+	app-arch/unzip"
+
+S="$WORKDIR"
+
+src_prepare() {
+	eapply "$FILESDIR/format.patch"
+	eapply_user
+
+	sed -e 's/\r//g' -i 972.txt
+	sed -e "/^ *THE DEVIL'S DICTIONARY/,/^End of Project Gutenberg's The Devil's Dictionary/!{w COPYING.gutenberg" -e 'd}' -i 972.txt
+	sed -e '/^\S/{: l;N;s/\n *\(.\)/ \1/g;t l}' -i 972.txt
+	sed -e "s/^\\([A-Zor .'?-]*[^,A-Zor .'?-]\\)/ \1/" -i 972.txt
+	sed -e '/^ /y/,/\a/' -i 972.txt
+}
+
+src_compile() {
+	head -n -6 972.txt | \
+		dictfmt -u "${SRC_URI% ->*}" \
+		-s "The Devil's Dictionary (2015-08-22 Project Gutenberg version)" \
+		--headword-separator " or " \
+		--columns 80 \
+		-h devils
+	sed -e 'y/\a/,/' -i devils.dict
+	dictzip devils.dict
+}
+
+src_install() {
+	dodoc COPYING.gutenberg
```

Comment:
What's this file? We normally don't install COPYING because it's redundant to LICENSE.

Reply:
It's the header info Project Gutenberg inserted into the raw file.  I'll take another look at it; it's not *quite* a public-domain license (it includes some things about not using the name "Project Gutenberg" in some cases), but neither does it seem like something we have in the LICENSE files, and I'm not entirely sure if it's the same across everything they provide.  If it does seem consistent, it may indeed be best to add it there instead.

Reply:
All right, the textual differences seem to just be different versions of the license, which I added to the repo.  We may want to involve the license project people to see if it could be added to one of the `@FREE-DOCUMENTS` groups.

The version I got from the website is admittedly different than that in either actual file; I'm reasonably confident that dictd-devils is fine with that as it includes a forwards-compatible clause in 1.E.1, but I'm not seeing one in the (quite old) version used by dictd-web1913.  I'm also not sure it's necessary as I *am* satisfying the "delete this license text" and can't find any "other references to Project Gutenberg" beyond the URL and the new license -- which is an explicit "out" it gives to get back to public domain.  Again, the license crew can figure that out better than I can, if we really want to dig into things.